src/System.Net.Primitives/tests/FunctionalTests/project.json

```diff
@@ -7,12 +7,16 @@
     "System.Runtime": "4.1.1-beta-24403-05",
     "System.Text.RegularExpressions": "4.2.0-beta-24403-05",
     "System.Threading.Tasks": "4.0.12-beta-24403-05",
+    "System.Net.Primitives": {
```

Comment:
I don't know why this would be required. The FunctionalTests should actually test the System.Net.Primitives package. 

Is this happening because xunit is using an older implementation of System.Net.Primitives?